### PR TITLE
[UK] Add `Beatmap Challenge Packs`

### PR DESCRIPTION
**Pull request by minusQuantumNeko**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/uk.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Вимоги для отримання медалей Beatmap Challenge Packs
+
+| Назва медалі | Вимога |
+| :-- | :-- |
+| Mappers' Guild Pack VII | Пройти найвищі складності усіх бітмап у збірці [Mappers' Guild Pack VII](https://osu.ppy.sh/beatmaps/packs/2034). Усі бітмапи в цій збірці мають найвищу складність яка знаходиться в діапазоні 5.00★–5.99★. |
+| Mappers' Guild Pack VIII | Пройти найвищі складності усіх бітмап у збірці [Mappers' Guild Pack VIII](https://osu.ppy.sh/beatmaps/packs/2035). Усі бітмапи в цій збірці мають найвищу складність яка знаходиться в діапазоні 6.00★–6.99★. |
+| Mappers' Guild Pack IX | Пройти найвищі складності усіх бітмап у збірці [Mappers' Guild Pack IX](https://osu.ppy.sh/beatmaps/packs/2036). Усі бітмапи в цій збірці мають найвищу складність яка знаходиться в діапазоні 7.00★–7.99★. |
+| Camellia II | Пройти усі бітмапи у збірці [Camellia Challenges Pack](https://osu.ppy.sh/beatmaps/packs/2053). Усі бітмапи в цій збірці мають найвищу складність `Expert` або вище. |
```
Comment (Pisapou) [inline wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/uk.md:8]:
```suggestion
| Camellia II | Пройти усі бітмапи у збірці [Camellia Challenges Pack](https://osu.ppy.sh/beatmaps/packs/2053). Усі бітмапи в цій збірці мають найвищу складність `Expert` або вище. |

```

**Comment by Pisapou** [comment]
Just remember that all wiki pages need a blank line at the end of it.

**Comment by Walavouchey** [comment]
it already has a trailing new line (ci would give an error if it didn't), github just doesn't show it

**Comment by Pisapou** [comment]
> it already has a trailing new line (ci would give an error if it didn't), github just doesn't show it

my bad sorry

**Review by sharytory — APPROVED** [review]
goog

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**